src/reporter/coverage-sonar-report.xsl

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet 
+    xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:x="http://www.jenitennison.com/xslt/xspec"
+    xmlns:test="http://www.jenitennison.com/xslt/unit-test" 
+    exclude-result-prefixes="xs"
+    version="2.0">
```

Comment:
```suggestion
    version="3.0">
```

This PR is scheduled for v2.0.0 where all `src/**/*.xsl` files should be `@version`=`3.0`.

(And you are free to use XSLT 3.0 features in this PR as far as it works on Saxon-HE 9.8.)

Reply:
This is still true.